app-shells/command_not_found/command_not_found-0.0.1~git.recipe

```diff
@@ -2,15 +2,19 @@ SUMMARY="A tool to find not-installed applications"
 DESCRIPTION="Command-not-found hooks into the command_not_found_handle  \
 to inform end users where to find an application if they don't have it. \
 It can also check for typos (a feature of zsh)."
-HOMEPAGE="https://github.com/jrabbit-cnf/"
+HOMEPAGE="https://github.com/jrabbit/haiku-cnf"
 COPYRIGHT="2011 Jrabbit"
 LICENSE="GNU GPL v2"
 REVISION="1"
-SOURCE_URI="git+https://github.com/jrabbit/haiku-cnf.git#87cd5d0c212e377cffa5d3e2a00c143fe751c6fb"
-PATCHES="command_not_found-0.0.1.patchset"
-DISABLE_SOURCE_PACKAGE="yes"
+srcGitRev="db1be8b57dedff781dc68caa46f403d88c5eedb9"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="a1ce4e6c8a7307c0d5c1e65a90b3d2a7ba9d16d9506a70aab1005129333d6121"
+SOURCE_FILENAME="haiku-cnf-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="haiku-cnf-$srcGitRev"
 
-ARCHITECTURES="?x86 ?x86_gcc2"
+PATCHES="command_not_found-0.0.1~git.patchset"
+
+ARCHITECTURES="?x86 ?x86_gcc2 x86_64"
```

Comment:
This is a pure python script. As such, it would work on any arch that has python.

```diff
-ARCHITECTURES="?x86 ?x86_gcc2 x86_64"
+ARCHITECTURES="any"